store/src/components/product/PopularProducts.js

```diff
@@ -34,44 +34,50 @@ function ProductList() {
   return (
     <MainContainer>
       <div style={{ marginTop: "2rem", marginBottom: "2rem" }}>
-        <HeaderSection headerText="popular products" />
+        <HeaderSection headerText="Popular Products" />
```

Comment:
No need to capitalize, since `HeaderSection` component already handles that.

Reply:
Ok, got it.